packages/dev/serializers/src/glTF/2.0/glTFMaterialExporter.ts

```diff
@@ -382,53 +225,85 @@ export class _GLTFMaterialExporter {
                 Tools.Warn(babylonStandardMaterial.name + ": glTF 2.0 does not support alpha mode: " + babylonStandardMaterial.alphaMode.toString());
             }
         }
-        if (babylonStandardMaterial.emissiveColor && !_GLTFMaterialExporter._FuzzyEquals(babylonStandardMaterial.emissiveColor, Color3.Black(), _GLTFMaterialExporter._Epsilon)) {
+
+        if (babylonStandardMaterial.emissiveColor && !babylonStandardMaterial.emissiveColor.equalsWithEpsilon(black, epsilon)) {
             material.emissiveFactor = babylonStandardMaterial.emissiveColor.asArray();
         }
 
         material.pbrMetallicRoughness = pbrMetallicRoughness;
-        _GLTFMaterialExporter._SetAlphaMode(material, babylonStandardMaterial);
+        SetAlphaMode(material, babylonStandardMaterial);
+
+        await this._finishMaterialAsync(material, babylonStandardMaterial, mimeType);
 
+        const materials = this._exporter._materials;
         materials.push(material);
-        materialMap[babylonStandardMaterial.uniqueId] = materials.length - 1;
+        return materials.length - 1;
+    }
+
+    private _convertToGLTFPBRMetallicRoughness(babylonStandardMaterial: StandardMaterial): IMaterialPbrMetallicRoughness {
+        // Defines a cubic bezier curve where x is specular power and y is roughness
+        const P0 = new Vector2(0, 1);
+        const P1 = new Vector2(0, 0.1);
+        const P2 = new Vector2(0, 0.1);
+        const P3 = new Vector2(1300, 0.1);
+
+        /**
+         * Given the control points, solve for x based on a given t for a cubic bezier curve
+         * @param t a value between 0 and 1
+         * @param p0 first control point
+         * @param p1 second control point
+         * @param p2 third control point
+         * @param p3 fourth control point
+         * @returns number result of cubic bezier curve at the specified t
+         */
+        function cubicBezierCurve(t: number, p0: number, p1: number, p2: number, p3: number): number {

```

Comment:
Why are these functions nested inside another function? Unless I missed it they don't capture any state from the outer context. Could they just be module level helper functions?